pybamm/expression_tree/symbol.py

```diff
@@ -195,6 +196,8 @@ class Symbol(anytree.NodeMixin):
         collector"}`).
     """
 
+    _print_name = None
```

Comment:
I didn't actually know you could do this here 😆 but for consistency let's put it in the `__init__`